app/assets/javascripts/select_change_label.js

```diff
@@ -0,0 +1,24 @@
+jQuery(function () {
+  $( document ).ready(function() {
+    $('[label_target]').each(function() {
+      change_label($(this));
+    })
+  });
+
+  $(document).on('change', '[label_target]', function () {
+    change_label($(this));
+  })
+
+  function change_label(element) {
+    var $id_label = element.attr('class_form').toLowerCase().concat('_', element.attr('label_target'));
+    var $label_target = $(`label[for="${$id_label}"]`);
```

Comment:
Esto del interpolado es "muy moderno" para los navegadores que tienen en los bancos, deberías ir por algo más _terrenal_ como:
```suggestion
    var $label_target = $('label[for="' + $id_label + '"]');
```

Reply:
dale